doc/rados/operations/erasure-code.rst

```diff
@@ -44,10 +45,18 @@ after the pool is created: a new pool with a different profile needs
 to be created and all objects from the previous pool moved to the new.
 
 The most important parameters of the profile are *K*, *M* and
-*crush-failure-domain* because they define the storage overhead and
-the data durability. For instance, if the desired architecture must
-sustain the loss of two racks with a storage overhead of 67% overhead,
-the following profile can be defined::
+*crush-failure-domain* because they define the storage overhead,
+the data durability, and the minimum hardware requirements.
+
+As a quick memory tool while selecting a profile, remember: up to
```

Comment:
> As a quick memory tool while selecting a profile, remember

I'd change this to:
"When selecting a profile, remember"

as-phrases (phrases that begin with "as") in English are notoriously slippery. This is especially true when "as" appears as the first word in the sentence. Moving this to the imperative mood and removing the present participle reduces the reader's cognitive load, which is good (especially in the case of this sentence, when we're about to ask the reader to juggle m, (m+k), and [(m+k)/k] ).